server/data/exampleApiClient.ts

```diff
@@ -0,0 +1,34 @@
+import { RestClient, asSystem, AuthenticationClient } from '@ministryofjustice/hmpps-rest-client'
+import config from '../config'
+import logger from '../../logger'
+
+export default class ExampleApiClient extends RestClient {
+  constructor(authenticationClient: AuthenticationClient) {
+    super('Example API', config.apis.exampleApi, logger, authenticationClient)
+  }
+
+  getCurrentTime() {
+    return this.get({ path: '/example/time' }, asSystem())
+  }
+
+  /**
+   * If wanting to make a call using the User's access token
+   *
+   * import { asUser } from '@ministryofjustice/hmpps-rest-client'
+   *
+   *   getCurrentTime(token: string) {
+   *     return this.get({ path: '/example/time' }, asUser(token))
+   *   }
+   */
+
+  /**
+   * If wanting to make a call using a System token,
+   * tied to a User (i.e. for backend auditing purposes)
+   *
+   * import { asUser } from '@ministryofjustice/hmpps-rest-client'
```

Comment:
little typo.. `asSystem`

Reply:
Totally rewritten in https://github.com/ministryofjustice/hmpps-template-typescript/pull/522/commits/fd8e10d61a3d8123bbe9007a92b0dc18f1557e0f - these were just drafts and I forgot to rewrite!